build.env

```diff
@@ -52,7 +52,7 @@ CSI_ATTACHER_VERSION=v3.4.0
 CSI_SNAPSHOTTER_VERSION=v5.0.1
 CSI_PROVISIONER_VERSION=v3.1.0
 CSI_RESIZER_VERSION=v1.4.0
-CSI_NODE_DRIVER_REGISTRAR_VERSION=v2.4.0
+CSI_NODE_DRIVER_REGISTRAR_VERSION=v2.5.1
```

Comment:
why update only the build.env, not the deployment yaml files?

Reply:
Huh.. missed to commit.. :( .updating